internal/characters/eula/burst.go

```diff
@@ -15,11 +15,12 @@ const burstHitmark = 100
 const lightfallHitmark = 35
 
 func init() {
-	burstFrames = frames.InitAbilSlice(122)
-	burstFrames[action.ActionDash] = 121
-	burstFrames[action.ActionJump] = 121
-	burstFrames[action.ActionSwap] = 121
-	burstFrames[action.ActionWalk] = 117
+	burstFrames = frames.InitAbilSlice(123) // Q -> E
```

Comment:
The tap and hold versions seem to have a difference in frame data (122 vs 123). Is this something that's okay to ignore?

Reply:
I think that we are ok with that, it's essentially the same cancel but recorded for completion.